doc/recipe/preprocessor.rst

```diff
@@ -1954,11 +1954,15 @@ along the longitude coordinate.
 Parameters:
     * `operator`: Operation to apply.
       See :ref:`stat_preprocs` for more details on supported statistics.
+    * `normalize_with_stats`: If given, do not return the statistics cube
```

Comment:
Would you be ok with calling this argument just `normalize` or `normalize_operator` instead? The abbreviation `stats` is for `statistics` is not immediately clear to me and writing `normalize_with_statistics` seems a bit long.

Reply:
Still feels weird to me to call that simply `normalize`, but the current one is also certainly not optimal. Will change to `normalize`.

Reply:
Thanks!